### DISCUSSION TITLE
"Resolve" option

### DISCUSSION THREAD

**Discussion by franciscoamado** [peripheryapp/periphery#313] [Ideas]
Hi there 👋🏻 

Thank you for building Periphery. 

I've been using it in a project and came up with the idea that it would be great if Periphery offered a command for cleaning or "resolving" the issues it found.

Have you thought about expanding Periphery in that direction?

**Comment by ileitch** [comment]
Hey,

Yeah I've thought about doing this, however I'm not sure it can be done in way that would actually be universally beneficial. I think it's important to review unused code with its surrounding context before removing it. And in the case of assign-only properties, removing the wrong property can actually cause bugs. There are also some instances where Periphery can produce false-positive results due to bugs in Swift, and having to undo those changes would be pretty annoying.

**Reply by franciscoamado** [reply]
I understand the problems this imposes. Even on my end, I wouldn't blindly trust such changes and would always have to go through a review first.
